corehq/apps/userreports/tests/test_view.py

```diff
@@ -173,16 +176,14 @@ def _build_report_and_view(self, request=HttpRequest()):
 
     @classmethod
     def tearDownClass(cls):
-        for case in cls.cases:
-            case.delete()
```

Comment:
I don't see a replacement for the case deletion logic here. Was deleting the cases only a proxy to drop the adapter tables? Will the cases still get deleted?

Reply:
My understanding is now that cases have been migrated to SQL, cleanup like this is no longer necessary, but it was necessary prior to that migration and when this test was initially written. And regardless of SQL or couch, I don't think deleting a case would drop a UCR table so I'm thinking cleaning up the UCR table was just overlooked initially.